types/gapi/index.d.ts

```diff
@@ -35,10 +35,20 @@ interface GoogleApiOAuth2TokenObject {
  */
 declare namespace gapi {
 
+    type LoadCallback = () => void | Function;
```

Comment:
If this is a callback, then `(...args:any[]) => void` should be your fallback. or just use `Function`. the union does not make much sense, since `Function` subsumes it.